### PR TITLE
Update main README

### LINKED ISSUE #562: Update main README

### PR DESCRIPTION
**Pull request by ioangatop**
Closes #562 

Improve and update the main README document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
 ```
 
-When used as a CLI tool, _`eva`_ supports configuration files (`.yaml`) as an argument to define its functionality.
-Native supported configs can be found at the [configs](https://github.com/kaiko-ai/eva/tree/main/configs) directory
-of the repo. Apart from cloning the repo, you can download the latest config folder as `.zip` from your browser from
-[here](https://download-directory.github.io/?url=https://github.com/kaiko-ai/eva/tree/main/configs). Alternatively,
-from a specific release the configs can be downloaded from the terminal as follows:
-```sh
-curl -LO https://github.com/kaiko-ai/eva/releases/download/0.0.1/configs.zip | unzip configs
+_`eva`_ employees [jsonargparse](https://jsonargparse.readthedocs.io/en/v4.31.0/) to
```
Comment (roman807) [inline README.md:73]: employees -> uses

File: README.md [hunk 2]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
+        batch_size: 32
 ```
-
-> [!NOTE] 
+</sub>
+</td>
+</tr>
+</table>
+
+So basically the `.yaml` file defines the functionality of _`eva`_
```
Comment (roman807) [inline README.md:173]: ~So basically the~ -> The

File: README.md [hunk 3]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
+of the repo, which can be both locally stored or remote.
+
+</details>
+
+## Quick Start
+
+We define two types of training: **online** and **offline**. Online fit involves
+using the backbone (also known as FM) to perform forward passes during the
+fitting process. In contrast, offline fit entails first generating embeddings
+from the backbone and then fitting the model using these embeddings, resulting in a faster evaluation.
```
Comment (roman807) [inline README.md:186]: Starting from 2n sentence:

```suggestion
While online fit uses the backbone (FM) to perform forward passes during the
fitting process, offline fit first generates embeddings
with the backbone and then fits the model using these embeddings as input, resulting in a faster evaluation.
```

File: README.md [hunk 4]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
+
+Here are some examples to get you started:
+
+- Performs a downstream offline **classification** evaluation of `DINO ViT-S/16`
```
Comment (roman807) [inline README.md:190]: Performs -> Perform (also in 198)

File: README.md [hunk 5]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
+on the `BACH` dataset with linear probing by first inferring the embeddings
+and performing 5 sequential fits:
```
Comment (roman807) [inline README.md:192]: ... and then performing...

File: README.md [hunk 6]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
+  ```
+
+- Performs a downstream online **segmentation** evaluation of `DINO ViT-S/16` on the
+`MoNuSAC` dataset with the `ConvDecoderMS` decoder:
+  ```sh
+  export DOWNLOAD_DATA=true
+  eva fit --config https://raw.githubusercontent.com/kaiko-ai/eva/main/configs/vision/dino_vit/online/monusac.yaml
+  ```
+
+For more examples, please look at the [configs](https://github.com/kaiko-ai/eva/tree/main/configs) and [tutorials](https://kaiko-ai.github.io/eva/dev/user-guide/advanced/replicate_evaluations/).
```
Comment (roman807) [inline README.md:205]: please look -> take a look

File: README.md [hunk 7]
```diff
@@ -68,43 +70,149 @@ _`eva`_ can be used directly from the terminal as a CLI tool as follows:
...
 ```
-
-> [!NOTE] 
+</sub>
+</td>
+</tr>
+</table>
+
+So basically the `.yaml` file defines the functionality of _`eva`_
+by parsing and translating its content to python objects directly.
```
Comment (roman807) [inline README.md:174]: capitalize python -> Python everywhere

**Review by roman807 — APPROVED** [review]
Nice update! Left a few comments for typos and alternative formulations

File: README.md (hunk 3 above)
Comment (ioangatop) [reply README.md:186]: nice, done

**Pull request merged**